identity-wallet/src/command.rs

```diff
@@ -4,14 +4,13 @@ use crate::state::persistence::save_state;
 use crate::state::{AppState, AppStateContainer};
 use futures::StreamExt;
 use itertools::Itertools;
-use log::error;
-use log::{debug, info, warn};
+use log::{debug, info, error};
 use std::time::Duration;
 use tauri::Manager;
 
 /// This function represents the root reducer of the application. It will delegate the state update to the reducers that
 /// are listening to the action.
-async fn reduce(state: AppState, action: Action) -> Result<AppState, AppError> {
+pub async fn reduce(state: AppState, action: Action) -> Result<AppState, AppError> {
```

Comment:
We don't know (yet) if we want to make this fully public, probably better to make it `pub(crate)` for now :)